product_attribute_priority/models/product.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses/.
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class ProductAttribute(models.Model):
+    _inherit = 'product.attribute'
+    _order = 'priority, name'
+
+    priority = fields.Integer(
+        string='Priority',
+        help='Define priority of exhibition in reports',
+        default=10,
+    )
+
+
+class ProductAttributeValue(models.Model):
+
+    _inherit = 'product.attribute.value'
+    _order = 'priority, name'
```

Comment:
I think it'll interesting if we could add another field (`value_sequence` or any). 
Yes because we have an issue on attribute values on website but also in some other views: 

If I have dimensions: `1 cm, 3 cm, 6 cm, 10 cm`  

> 10  is in second position after 1 cm, whereas it should be last

Then we have `_order = 'priority, value_sequence, name'`.
With this settings, order of the values could be defined for those particular values.

What do you think ? Do you accept a PR on your PR ?

Thanks